src/ReplayingMessages/ReplayResult.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EventSauce\EventSourcing\ReplayingMessages;
+
+use EventSauce\EventSourcing\PaginationCursor;
+
+class ReplayResult
+{
+    public function __construct(private int $messagesHandled, private ?PaginationCursor $cursor)
```

Comment:
`readonly` ? Or... are we sure we can use these features?

Reply:
Readonly is php 8.1, we're currently supporting 8.0+